Vignettes for dm and ae domains

ShiyuC opened the pull request:
closes #116 

- Create vignettes for DM and AE domains
- Update `study_ct` file

ShiyuC committed:
Vignette for dm domain

rammprasad commented on vignettes/demographics_domain.Rmd:
````diff
@@ -0,0 +1,420 @@
+)
+```
+
+## Create reference dates configuration file {#refdates}
````
Can you add the column definitions here? It will be easier to the user to understand. Ideally, the user should understand how to prepare this file by looking at this Vignette

ShiyuC replied:
Added definitions of each column

rammprasad commented on vignettes/demographics_domain.Rmd:
````diff
@@ -0,0 +1,420 @@
+on if the variable has controlled terminologies associated.
+
+```{r eval=TRUE}
+dm <-
+  # Map AGE using assign_no_ct
+  assign_no_ct(
+    raw_dat = dm_raw,
````
I am gettting a CT error while execuring this chunk. Is it working for you?

ShiyuC replied:
It works fine for me. Could you double check if the study_ct file you loaded contains 138 observations?

rammprasad commented on vignettes/demographics_domain.Rmd:
```diff
@@ -0,0 +1,420 @@
+  # Map DTHDTC using oak_cal_ref_dates
+  oak_cal_ref_dates(ds_in = .,
+                    der_var = "DTHDTC",
+                    min_max = "max",
```
I think for death date, it has to be min.

ShiyuC replied:
updated

rammprasad commented on vignettes/demographics_domain.Rmd:
```diff
@@ -0,0 +1,420 @@
+  # Derive RFSTDTC using oak_cal_ref_dates
+  oak_cal_ref_dates(ds_in = .,
+                    der_var = "RFSTDTC",
+                    min_max = "min",
```
It will be beneficial to explain how to choose min or max for each function call. Explain the derivation of RFSTDTC and explain the parameters, like min or max. Lets just do it for one min and one max examples and then rest doesn't need it.

ShiyuC replied:
I added separate descriptions for RFSTDTC and RFENDTC explaining the selection of min/max. Please review again and see how it looks to you

rammprasad reviewed (changes requested):
GOod job,. Looks awesome. Minor changes.

ShiyuC committed:
Update per comments

ShiyuC committed:
Vignette for AE domain; update description

ShiyuC committed:
run styler

ShiyuC committed:
events domain styler

ShiyuC committed:
update title of the rmd file to knit rmd

rammprasad committed:
Updated DM vignette

rammprasad committed:
Merge branch '116-docs-aedm' of https://github.com/pharmaverse/sdtm.oak into 116-docs-aedm

ShiyuC committed:
small tweaks

ShiyuC committed:
run styler

ShiyuC committed:
main updates

Merge remote-tracking branch 'origin/main' into 116-docs-aedm
# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

ShiyuC committed:
Update comments in dm snippet
